InvenTree/stock/api.py

```diff
@@ -830,6 +830,16 @@ def filter_queryset(self, queryset):
                     queryset = queryset.filter(StockItem.EXPIRED_FILTER)
                 else:
                     queryset = queryset.exclude(StockItem.EXPIRED_FILTER)
+            # Filter by 'expiry date'
+            expired_date_lte = params.get('expiry_date_lte', None)
+            if expired_date_lte is not None:
+                date_lte = datetime.fromtimestamp(int(expired_date_lte) / 1e3)
```

Comment:
I would very much prefer passing UTC ISO dates here instead of timestamps. Requires less computation on the client side